tests/test_bears/AspectTestBear.py

```diff
@@ -0,0 +1,29 @@
+from coalib.bearlib.aspects import Root
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Result import Result
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+
+
+class AspectTestBear(
+        LocalBear,
+        aspects={
+            'detect': [
+                Root.Redundancy.UnusedVariable.UnusedGlobalVariable,
+                Root.Redundancy.UnusedVariable.UnusedLocalVariable,
+            ]},
+        languages=['Python']):
```

Comment:
put the `):` on the next line, add a trailing `,`, and dont have a blank line between the `):` and the first line of code.